conf/test_full.config

```diff
@@ -17,8 +17,7 @@ params {
     // Input data for full size test
     input = "${baseDir}/tests/csv/3.0/test_full_data.csv"
 
-    wes   = true
+    genome = "GATK.GRCh38"
```

Comment:
do we want to add some variant calling tools here as well?

I'm not happy doing that in `test.config` because for me it should be as minimal as possible.
But it makes perfect sense having some here

Reply:
yes, totally right, I forgot about adding tools!